examples/spatialdata_demo.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
<p>this line should render the plotting directly right? so why are the images embedded? I would prefer to have the image directly rendered from the code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/napari-spatialdata/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>